packaging/tags.py

```diff
@@ -539,20 +547,19 @@ def _parse_glibc_version(version_str):
     return (int(m.group("major")), int(m.group("minor")))
 
 
-def _have_compatible_glibc(tag_major, tag_minor):
-    # type: (int, int) -> bool
-    version_str = _glibc_version_string()
-    if version_str is None:
-        return False
-    return _check_glibc_version(version_str, tag_major, tag_minor)
```

Comment:
`_have_compatible_glibc` did two things, so refactored it out into the component actions